sample-code/src/orchestration.ts

```diff
@@ -39,6 +41,32 @@ export async function orchestrationChatCompletion(): Promise<OrchestrationRespon
   return result;
 }
 
+/**
+ * Ask GPT 3.5 through the orchestration service about SAP Cloud SDK with streaming.
+ * @param controller - The abort controller.
+ * @returns The response from Azure OpenAI containing the response content.
+ */
+export async function chatCompletionStream(
+  controller: AbortController
+): Promise<
+  OrchestrationChatCompletionStreamResponse<OrchestrationChatCompletionStreamChunkResponse>
+> {
+  const orchestrationClient = new OrchestrationClient({
+    // define the language model to be used
+    llm: {
+      model_name: 'gpt-4o',
+      model_params: {}
+    },
+    // define the prompt
+    templating: {
+      template: [{ role: 'user', content: 'Give me a short introduction of SAP Cloud SDK.' }]
+    }
+  });
+
+  const response = await orchestrationClient.stream(undefined, controller);
```

Comment:
currently, this API doesn't allow users to modify the stream options of orchestration. Eventually, this should be possible, so if not done with this BLI please create a follow-up so that we don't forget 😉

Reply:
Also I checked with Dennis, if you provide streaming options, but call a non-streaming API, they are simply ignored.

Reply:
The client expects `OrchestrationModuleConfig`, but it's not part of the module config, but on `orchestration_config` instead. So I wouldn't know how it can be customized currently..

> a non-streaming API
what is that?

Reply:
> The client expects OrchestrationModuleConfig, but it's not part of the module config, but on orchestration_config instead. So I wouldn't know how it can be customized currently..

Yeah right, I just saw the options in the orchestration documentation, I would assume they end up in the spec at which point we will re-generate, then the config types will include the streaming options.

> what is that? Do you mean a model that doesn't support streaming?
No, rather if you send the config including streaming options but against a regular non-streaming chat-compltion API, e.g. you didn't set `streaming: true`

Reply:
yeah, so with the current API we would either have to make it part of the prompt, or a new, separate parameter, or somehow else allow the user to pass this.

Okay, so you mean sending e.g.:
```json
{ "orchestration_config": 
  {
     "stream": false,
     "stream_options": {
         "chunk_size": 20
      }
  // ...
  }
}
```
?

Reply:
Yes exactly.
You can still pass streaming options as part of the options object, but you would have to typecast it to "any", as an ugly workaround.

Reply:
which options object? I don't see how currently the stream options can be passed at all..

Reply:
In the initialization of the client.

Reply:
I might be blind, but can you give an example code snippet? I don't see how this works..